SimplCalc/ViewModels/MainWindowViewModel.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Collections.ObjectModel;
+using System.Windows.Input;
+using SimplCalc.RelayCommands;
+using SimplCalc.Model;
+using System.ComponentModel;
+
+namespace SimplCalc.ViewModels
+{
+    class MainWindowViewModel: INotifyPropertyChanged
```

Comment:
Also keep INotifyPropertyChanged and RelayCommand class into a separate class called ViewModelBase for clarity but this is acceptable